finagle-http/src/test/scala/com/twitter/finagle/http/AbstractEndToEndTest.scala

```diff
@@ -916,4 +918,138 @@ abstract class AbstractEndToEndTest extends FunSuite
     await(server.close())
     await(client.close())
   }
+
+  testIfImplemented(NoBodyMessage)(
```

Comment:
Yeah, you're right. It's nice to have small tests! Hope my modification is what you suggested :)